.env.example

```diff
@@ -0,0 +1,11 @@
+TELEGRAM2PHOTOPRISM_TELEGRAM_ACCESS_TOKEN=your_telegram_access_token
+TELEGRAM2PHOTOPRISM_TELEGRAM_CHAT_ID=your_telegram_chat_id
+TELEGRAM2PHOTOPRISM_BOT_API_SERVER=your_photoprism_url
```

Comment:
It is not a good idea to rewrite default parameters inside .env.example. It would be better to write them as a comment:
```
; TELEGRAM2PHOTOPRISM_BOT_API_SERVER=https://api.telegram.org
```